pages/contributing/metadata_guidelines.md

```diff
@@ -3,107 +3,145 @@ title: Metadata Guidelines
 summary: Descriptions of different metadata schemas used to describe various resources and pages in RSQKit.
 ---
 
-In this page we describe the metadata schemas  used to describe various resources and pages in the RSQKit (adapted from the [RDMKit page metadata guide](https://rdmkit.elixir-europe.org/page_metadata)).
+This page explains the metadata that can be used to describe various resources and pages in RSQKit. 
+These guidelines have been adapted from the ELIXIR Toolkit theme's [page metadata guide](https://elixir-belgium.github.io/elixir-toolkit-theme/page_mechanics)
+and [tools and resources guide](https://elixir-belgium.github.io/elixir-toolkit-theme/resource_table).
 
-## Metadata attributes of a page
-When creating a new page in RSQKit, the following metadata attributes may be edited:
+## Contributor metadata
 
-* `title`: Specify here the title of the page. This wil be the H1 title (replacing the top level title using the # in markdown )
+Contributors are listed in the [contributors file (`_data/CONTRIBUTORS.yml`)](https://github.com/EVERSE-ResearchSoftware/RSQKit/blob/main/_data/CONTRIBUTORS.yml)
+starting with the contributor's full name, followed by the following optional metadata/attributes:
 
-* `summary`: Using this attribute it is possible to specify a summary which will be displayed under the title of the page. This summary will also be used as description of your page when the page is tagged.
+* `git`: contributor's GitHub id (serving as an identifier together with contributor's full name).
+* `orcid`: contributor's ORCID id.
+* `role`: role of the contributor in the RSQKit (e.g. `editor`).
```

Comment:
perhaps we need to say for any other role than editor this can be blank

Reply:
I say that all parameters apart from the name are optional but I get your point - perhaps it is better to emphasise that.

Reply:
That's another thing I wanted to tlk to you about - add more explicit roles - contributor, author, editor etc. to make it easier to explain what different contribution type are there. Having multiple roles would be great but the ETT does not support this.